_maps_2024/29-aki.md

```diff
@@ -0,0 +1,8 @@
+---
+day: 29
+creator: Akira Di Sandro
+title: Overture Maps
+thumbnail: /assets/thumbnails/2024/29-Aki-Overture.html
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day29.R
+---
+
```

Comment:
**suggestion:** You'll still have to provide a thumbnail image (something to show on the front page -- note that I refer to a PNG file that doesn't yet exist below, so you'd have to create it), but the best thing to do might be to create that image, and then link to the HTML page for the interactive bit:
```suggestion
thumbnail: /assets/thumbnails/2024/29-Aki-Overture.png
source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day29.R
---

View the [interactive map](assets/thumbnails/2024/29-Aki-Overture.html) of building classifications in Firenze, Italy.

![Screenshot of map](assets/thumbnails/2024/29-Aki-Overture.png)
```

Reply:
thank you! Just added the png as well